source/nodejs/adaptivecards-designer/src/open-sample-dialog.ts

```diff
@@ -9,9 +9,20 @@ class CatalogueItem {
 
     constructor(readonly entry: CatalogueEntry) { }
 
+    private static _id = 0;
+    private static _getNewItemId(prefix: string): string {
```

Comment:
Style consistency: underscores are only used for private fields. Methods shouldn't use them.